CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ set_package_properties(OpenSSL PROPERTIES
     TYPE REQUIRED
 )
 
-set(Python_ADDITIONAL_VERSIONS 2.7)
+set(Python_ADDITIONAL_VERSIONS 3.7)
```

Comment:
This should be bumped in 05b4ded.

Reply:
This diff is outdated--the Python version is no longer supplied to the CMake finder.

Reply:
Ah, good.  I didn't see it updated but also didn't see it was gone!